backtrace-library/src/main/java/backtraceio/library/common/DeviceAttributesHelper.java

```diff
@@ -187,20 +186,23 @@ private WifiStatus getWifiStatus() {
      * @return is power saving mode activated
      */
     // TODO: replace bool to enum
+    /*
     private boolean isPowerSavingMode() {
         if (Build.VERSION.SDK_INT < 21) {
             return false;
         }
         PowerManager powerManager = (PowerManager) this.context.getSystemService(Context
                 .POWER_SERVICE);
         return powerManager.isPowerSaveMode();
-    }
+    }*/
 
+    // Try removing this since it's not the correct way to do this
```

Comment:
If we trust the new method from `BatteryStateHelper` to do this now, we can just remove this, no need to comment it out